variants/canaryone/platformio.ini

```diff
@@ -1,13 +1,13 @@
-; First prototype eink/nrf52840/sx1262 device
-[env:canary]
+; Public Beta oled/nrf52840/sx1262 device
+[env:canaryone]
 extends = nrf52840_base
-board = canary
+board = canaryone
 debug_tool = jlink
 
 # add -DCFG_SYSVIEW if you want to use the Segger systemview tool for OS profiling.
-build_flags = ${nrf52840_base.build_flags} -Ivariants/canary
+build_flags = ${nrf52840_base.build_flags} -Ivariants/canaryone
```

Comment:
@lolsborn maybe I am just a noob, but why does the `-I` flag not have a space... e.g. `-I variants/canaryone`

Reply:
The space is not necessary and I _think_ it gets preserved and passed to C so is left out a lot of times, but it seems to be present in all of the other platform.ini files and doesn't seem to break things so I'm fine having it.